### PR TITLE
support multilabel

### PR DESCRIPTION
**Pull request by kexinhuang12345**

### CODE REVIEW

File: examples/gnn_node.py [hunk 1]
```diff
@@ -149,6 +162,9 @@ def test(loader: NeighborLoader) -> np.ndarray:
         if task.task_type == TaskType.BINARY_CLASSIFICATION:
             pred = torch.sigmoid(pred)
 
+        if task.task_type == TaskType.MULTILABEL_CLASSIFICATION:
+            pred = torch.sigmoid(pred)
```
Comment (weihua916) [inline examples/gnn_node.py:166]: nit:
```python
if task.task_type in [TaskType.BINARY_CLASSIFICATION, TaskType.MULTILABEL_CLASSIFICATION]:
```

File: examples/gnn_node.py [hunk 2]
```diff
@@ -120,7 +132,8 @@ def train() -> float:
             task.entity_table,
         )
         pred = pred.view(-1) if pred.size(1) == 1 else pred
-        loss = loss_fn(pred, batch[entity_table].y)
+
+        loss = loss_fn(pred.float(), batch[entity_table].y.float())
```
Comment (weihua916) [inline examples/gnn_node.py:136]: Do you mind changing `test_node_nn.py` accordingly to test your changes in this `gnn_node.py`?

File: relbench/external/graph.py [hunk 3]
```diff
@@ -173,7 +175,12 @@ def get_node_train_table_input(
         target_type = float
         if task.task_type == "multiclass_classification":
             target_type = int
-        target = torch.from_numpy(table.df[task.target_col].values.astype(target_type))
+        if task.task_type == TaskType.MULTILABEL_CLASSIFICATION:
+            target = torch.from_numpy(np.stack(table.df[task.target_col].values))
```
Comment (weihua916) [inline relbench/external/graph.py:179]: add `astype(target_type)`?

**Review by weihua916 — APPROVED** [review]
Thanks! Can we modify the multi-label metrics as a follow-up?

File: examples/gnn_node.py [hunk 4]
```diff
@@ -41,7 +41,7 @@
 root_dir = "./data"
 
 # TODO: remove process=True once correct data/task is uploaded.
-dataset: RelBenchDataset = get_dataset(name=args.dataset, process=True)
+dataset: RelBenchDataset = get_dataset(name=args.dataset, process=False)
```
Comment (weihua916) [inline examples/gnn_node.py:44]: Why do we make `process = False` here?
